packages/block-editor/src/components/plain-text/stories/index.story.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Internal dependencies
+ */
+import PlainText from '..';
```

Comment:
By convention, I think the three dependencies should be listed in the following order:

- External dependencies
- WordPress dependencies
- Internal dependencies